GetIntoTeachingApi/Services/CrmService.cs

```diff
@@ -16,6 +16,12 @@ public enum PrivacyPolicyType
             Web = 222750001,
         }
 
+        public enum CandidateStatus
+        {
+            Active = 0,
+            Inactive = 1,

```

Comment:
you don't need the '= 1' it increments by each line as default, I know you love saving code ;)